templates/bot/js/command-and-response/README.md

```diff
@@ -49,148 +47,144 @@ The following files provide the business logic for command and response bot. The
-
-To add the notification feature:
-
-1. Go to `bot\src\internal\initialize.js`
-2. Update your `conversationBot` initialization to enable notification feature:
-    ![enable-notification](https://user-images.githubusercontent.com/10163840/165462039-12bd4f61-3fc2-4fc8-8910-6a4b1e138626.png)
-3. To quickly add a sample notification triggered by a HTTP request, you can add the following sample code in `bot\src\index.js`:
-
-    ```javascript
-    server.post("/api/notification", async (req, res) => {
-      for (const target of await commandBot.notification.installations()) {
-        await target.sendMessage("This is a sample notification message");
-      }
-    
-      res.json({});
-    });
-
-4. Uninstall your previous bot installation from Teams, and press `F5` to start your application.
-5. Send a notification to the bot installation targets (channel/group chat/personal chat) by using a your favorite tool to send a HTTP POST request to `http://localhost:3978/api/notification`.
-
-To learn more, refer to [the notification document](https://aka.ms/teamsfx-notification).
-
-## Add adaptive card actions
-The Adaptive Card action handler feature enables the app to respond to adaptive card actions that triggered by end users to complete a sequential workflow. When user gets an Adaptive Card, it can provide one or more buttons in the card to ask for user's input, do something like calling some APIs, and then send another adaptive card in conversation to response to the card action.
-
-To add adaptive card actions to command bot, you can follow the steps [here](https://aka.ms/teamsfx-card-action-response#add-more-card-actions).
-
-## Accessing Microsoft Graph
-
-If you are responding to a command that needs access to Microsoft Graph, you can leverage single sign on to leverage the logged-in Teams user token to access their Microsoft Graph data. Read more about how Teams Toolkit can help you [add SSO](https://aka.ms/teamsfx-add-sso) to your application.
-
-## Connect to existing APIs
+```javascript
+const doSomethingCard = require("./adaptiveCards/doSomethingCommandResponse.json");
+const { MessageBuilder } = require("@microsoft/teamsfx");
+
+class DoSomethingCommandHandler implements TeamsFxBotCommandHandler {
+    triggerPatterns: TriggerPatterns = "doSomething";
+
+    async handleCommandReceived(
+        context: TurnContext,
+        message: CommandMessage
+    ): Promise<string | Partial<Activity>> {
+        // verify the command arguments which are received from the client if needed.
+        console.log(`Bot received message: ${message.text}`);
+
+        const cardData = {
+          title: "doSomething command is added",
+          body: "Congratulations! You have responded to doSomething command",
+        };
+
+        return MessageBuilder.attachAdaptiveCard(helloWorldCard, cardData);
```

Comment:
Let's replace the following code line:
```
return MessageBuilder.attachAdaptiveCard(helloWorldCard, cardData);
```
with
```jsvascript
const cardJson = AdaptiveCards.declare(doSomethingCard).render(cardData);
return MessageFactory.attachment(CardFactory.adaptiveCard(cardJson));
```

Reply:
Done